poetry/console/commands/add.py

```diff
@@ -89,13 +89,18 @@ def handle(self):
         if section not in poetry_content:
             poetry_content[section] = {}
 
-        packages = self.remove_existing_packages_from_input(
+        existing_packages = self.get_existing_packages_from_input(
             packages, poetry_content, section
         )
 
+        if existing_packages:
```

Comment:
Looking good. Do you reckon it might be better to move the message to the end so that it is more .. visible to the user rather than lost somewhere at the top? Granted people might not add a lot of packages, but if they added one with a lot dependencies the install log could cover up the message.

Reply:
I prefer having this message at the beginning,  because it saying something like "Ok, I will do it for you, but ..." The paragraph is large enough to be an eye-catcher. So when someone start the process they should recognize that it looks different than normal.

Putting it at the end, I would find myself always asking: "Hm, did it install everything fine".

But at the end it - as always - depends on how people are used to read/scan messages :)

Reply:
Typically terminals scroll messages up, so I'd expect this to be more in the summary of results from the operation. However, since this is not a major issue, I am not going to block on this.